composer.json

```
@@ -5,10 +5,10 @@
     "type": "library",
     "license": "MIT",
     "require": {
-        "guzzlehttp/guzzle": "^6.3"
+        "guzzlehttp/guzzle": "^6.3 || ^7.0"
```

Please remove the `6.3` version and use the latest version of the `guzzlehttp/guzzle` package 
```suggestion
        "guzzlehttp/guzzle": "^7.5"
```